### PR TITLE
Auto package Yolov5

### PR DESCRIPTION
**Pull request by KSGulin**
Adds supporting files to package Yolov5 into a pip installable, while supporting original CLI commands. 

`build_wheel.sh` is the driver for the packaging and will produce a `.whl` file in `./dist/`

The package structure follows the original Yolov5 directory structure, but interface for run scripts (train, val, and export) is not provided. Some example imports:
```
from yolov5.models.common import Conv
from yolov5.utils.activations import SiLU
```

### CODE REVIEW

**Review by bfineran — COMMENTED** [review]
looks good overall, in the PR description can you provide some sample imports of from this package?

File: build_wheel.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+python3 -m pip install --upgrade build
+mkdir yolov5
+cp -r data models models_v5.0 utils .pre-commit-config.yaml $(ls *.py) requirements.txt yolov5/
+grep --include=\*.py -rnl 'yolov5/' -e "from models" | xargs -i@ sed -i 's/from models/from yolov5.models/g' @
+grep --include=\*.py -rnl 'yolov5/' -e "from utils" | xargs -i@ sed -i 's/from utils/from yolov5.utils/g' @
```
Comment (bfineran) [inline build_wheel.sh:5]: nice

**Review by bfineran — COMMENTED** [review]
LGTM. let's also add an `NM_INTEGRATED=True` property somewhere in the package so we can test for it here: https://github.com/neuralmagic/sparseml/blob/39ad93a80e7515c8edc2cc95d1d91b71ae8f3700/src/sparseml/transformers/__init__.py#L104
(not super necessary since there aren't any other releases but I think this would be good for consistency and some future-proofing)

**Comment by KSGulin** [comment]
Good call @bfineran. Added it to `utils/general.py` and added a line to `build_wheel.sh` to import it.

**Review by bfineran — APPROVED** [review]

**Review by rahul-tuli — APPROVED** [review]

**Comment by rahul-tuli** [comment]
Good to land, pending failing tests! nice work on this @KSGulin

**Pull request merged**